documentation/collectionruleexamples.md

```diff
@@ -0,0 +1,260 @@
+# Collection Rule Examples
+
+The following examples provide sample scenarios for using a collection rule. These templates can be copied directly into your configuration file with minimal adjustments to work with your application (for more information on configuring an egress provider, see [egress providers](./configuration.md#egress-configuration)), or they can be adjusted for your specific use-case. [Learn more about configuring collection rules](collectionrules.md).
+
+## Collect Trace - Startup (`Startup` Trigger)
+
+### JSON
+
+```json
+{
+  "AssemblyLoadTraceOnStartup": {
+    "Trigger": {
+      "Type": "Startup"
+    },
+    "Actions": [
+      {
+        "Type": "CollectTrace",
```

Comment:
You'll want something like:
`
"Actions": [
    {
        "Type": "CollectTrace",
        "Settings": {
            "Duration": "00:00:15",
            "Egress": "file",
            "Providers": [
                {
                    "Name": "Microsoft-Windows-DotNETRuntime",
                    "Keywords": "0x8",
                    "EventLevel": "Informational"
                }],